recipes/compas_slicer/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "compas_slicer" %}
+{% set version = "0.3.3" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/compas_slicer-{{ version }}.tar.gz
+  sha256: 6d7cc6c1a84ac28f32b33b8ad6d78d1a3f7e7b024093dd4a156594c13f428c4b
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
```

Comment:
I suspect that this one can be `noarch: python`. Can you check if there are compiled extensions of something that is python version/platform specific that would prevent us from using noarch here?

Reply:
you're totally right! updated it to `noarch`